subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y)
```

Comment:
Missing colon on line 1 and missing blank new line at end of file

Reply:
solved